src/index.html

```diff
@@ -9,11 +9,59 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Moyo</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <header>
+    <a href="#" class="logo-box">
+      <img src="./images/logo.png" alt="MOYO">
```

Comment:
Screen reader reads this like `moyo`. What does it mean?
`Product photo` will be better.